cli/flags.rs

```diff
@@ -83,88 +98,76 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
       SubCommand::with_name("version")
         .setting(AppSettings::DisableVersion)
         .about("Print the version")
-        .long_about(
-          "
-Print current version of Deno.
+        .long_about("Print current version of Deno.
 
 Includes versions of Deno, V8 JavaScript Engine, and the TypeScript
-compiler.
-",
+compiler.",
         ),
     ).subcommand(
       SubCommand::with_name("fetch")
         .setting(AppSettings::DisableVersion)
         .about("Fetch the dependencies")
         .long_about(
-          "
-Fetch and compile remote dependencies recursively.
+          "Fetch and compile remote dependencies recursively.
 
 Downloads all statically imported scripts and save them in local
 cache, without running the code. No future import network requests
 would be made unless --reload is specified.
 
   # Downloads all dependencies
   deno fetch https://deno.land/std/http/file_server.ts
+
   # Once cached, static imports no longer send network requests
-  deno run -A https://deno.land/std/http/file_server.ts
-",
+  deno run -A https://deno.land/std/http/file_server.ts",
         ).arg(Arg::with_name("file").takes_value(true).required(true)),
     ).subcommand(
       SubCommand::with_name("types")
         .setting(AppSettings::DisableVersion)
         .about("Print runtime TypeScript declarations")
-        .long_about(
-          "
-Print runtime TypeScript declarations.
-
-The declaration file could be saved and used for typing information.
+        .long_about("Print runtime TypeScript declarations.
 
   deno types > lib.deno_runtime.d.ts
-",
+
+The declaration file could be saved and used for typing information.",
         ),
     ).subcommand(
       SubCommand::with_name("info")
         .setting(AppSettings::DisableVersion)
         .about("Show source file related info")
-        .long_about(
-          "
-Show source file related info.
+        .long_about("Show source file related info.
 
-The following information is shown:
+  deno info https://deno.land/std@v0.6/http/file_server.ts
 
-local:     local path of the file.
-type:      type of script (e.g. JavaScript/TypeScript/JSON)
-compiled:  (TypeScript only) shown local path of compiled source code.
-map:       (TypeScript only) shown local path of source map.
-deps:      dependency tree of the source file.
+The following information is shown:
 
-  deno info myfile.ts
-",
+  local: local path of the file.
+  type: JavaScript,TypeScript, or JSON.
+  compiled: TypeScript only. shown local path of compiled source code.
+  map: TypeScript only. shown local path of source map.
+  deps: dependency tree of the source file.",
```

Comment:
nit: capitalization, also aligned whitespace in previous version was more readable IMHO